RecoLocalCalo/HcalRecAlgos/interface/PulseShapeFitOOTPileupCorrection.h

```diff
@@ -120,14 +121,14 @@ class PulseShapeFitOOTPileupCorrection
     const HcalPulseShapes::Shape* currentPulseShape_=nullptr;
     void setChi2Term( bool isHPD );
 
-    void setPulseShapeTemplate  (const HcalPulseShapes::Shape& ps, bool isHPD);
-    void resetPulseShapeTemplate(const HcalPulseShapes::Shape& ps);
+    void setPulseShapeTemplate  (const HcalPulseShapes::Shape& ps, bool isHPD, unsigned nSamples_);
```

Comment:
please use consistent style for variable names:
- local variables without underscore
- trailing underscore reserved for data members of the class